experimental/dds/tree2/src/feature-libraries/optional-field/optionalField.ts

```diff
@@ -445,29 +444,24 @@ export const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = {
 		return rebased;
 	},
 
-	amendRebase: (
-		change: OptionalChangeset,
-		overTagged: TaggedChange<OptionalChangeset>,
-		rebaseChild: NodeChangeRebaser,
-	) => {
-		const amended = { ...change };
-		if (change.childChanges !== undefined) {
-			const overChildChanges = new ChildChangeMap<NodeChangeset>();
-			for (const [id, overChange] of overTagged?.change.childChanges ?? []) {
-				overChildChanges.set(id, overChange);
-			}
+	prune: (change: OptionalChangeset, pruneChild: NodeChangePruner): OptionalChangeset => {
+		if (change.childChanges === undefined) {
+			return change;
+		}
 
-			const childChanges: typeof change.childChanges = [];
-			for (const [id, childChange] of change.childChanges) {
-				const rebasedChange = rebaseChild(childChange, overChildChanges.get(id));
-				if (rebasedChange !== undefined) {
-					childChanges.push([id, rebasedChange]);
-				}
-			}
+		const prunedChange: OptionalChangeset = {};
+		if (change.fieldChange !== undefined) {
+			prunedChange.fieldChange = change.fieldChange;
```

Comment:
There's pruning to do in `fieldChange.newContent.changes`. This makes the above return invalid in some cases.